contracts/p0/BasketHandler.sol

```diff
@@ -170,7 +170,7 @@ contract BasketHandlerP0 is ComponentP0, IBasketHandler {
 
         require(
             main.hasRole(OWNER, _msgSender()) ||
-                (status() == CollateralStatus.DISABLED && !main.pausedOrFrozen()),
+                (status() == CollateralStatus.DISABLED && !main.frozen()),
```

Comment:
I wonder if we should keep this as `pausedOrFrozen()`...if the system is paused why should we support changing the basket? Recall that at least while it is DISABLED people can get the balance of DISABLED collateral token out, while if we change the basket then they won't get anything, and we won't have any of the token replacing it yet until we become unpaused and trade.